Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs

```diff
@@ -26,6 +28,55 @@ namespace Microsoft.MixedReality.Toolkit.Tests
 {
     public class PlayModeTestUtilities
     {
+        const string playModeTestSceneName = "PlayModeTestScene";
+
+        /// <summary>
+        /// Creates a play mode test scene, creates an MRTK instance, initializes playspace.
+        /// </summary>
+        public static void Setup()
+        {
+            if (EditorApplication.isPlaying)
+            {
+                Scene playModeTestScene = SceneManager.GetSceneByName(playModeTestSceneName);
+                if (!playModeTestScene.IsValid() || !playModeTestScene.isLoaded)
+                {
+                    playModeTestScene = SceneManager.CreateScene(playModeTestSceneName);
+                    SceneManager.SetActiveScene(playModeTestScene);
+                }
+            }
+            else
+            {
+                // At the very start of testing, playmode may not be active. In this case, create a fresh scene
+                Scene playModeTestScene = EditorSceneManager.NewScene(NewSceneSetup.DefaultGameObjects, NewSceneMode.Additive);
+                EditorSceneManager.SetActiveScene(playModeTestScene);
+            }
+
+            // Change our render settings so we can see what's going on more clearly
```

Comment:
I've actually removed RnderSettings.skybox = null from all my recent tests because it's actually *harder* to see the black boxes we instantiateover a black background. Can you remove it?